[2024-09-19T18:13:45.308+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-09-17T00:00:00+00:00 [queued]>
[2024-09-19T18:13:47.543+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schema_dag.submit_schemas scheduled__2024-09-17T00:00:00+00:00 [queued]>
[2024-09-19T18:13:47.880+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-09-19T18:13:51.867+0000] {taskinstance.py:1380} INFO - Executing <Task(PinotSchemaSubmitOperator): submit_schemas> on 2024-09-17 00:00:00+00:00
[2024-09-19T18:13:52.136+0000] {standard_task_runner.py:57} INFO - Started process 441 to run task
[2024-09-19T18:13:52.331+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'schema_dag', 'submit_schemas', 'scheduled__2024-09-17T00:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/schema_dag.py', '--cfg-path', '/tmp/tmpf_zidl7a']
[2024-09-19T18:13:52.409+0000] {standard_task_runner.py:85} INFO - Job 35: Subtask submit_schemas
[2024-09-19T18:14:13.693+0000] {task_command.py:415} INFO - Running <TaskInstance: schema_dag.submit_schemas scheduled__2024-09-17T00:00:00+00:00 [running]> on host 48913e9acee9
[2024-09-19T18:14:38.979+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='codewithyu' AIRFLOW_CTX_DAG_ID='schema_dag' AIRFLOW_CTX_TASK_ID='submit_schemas' AIRFLOW_CTX_EXECUTION_DATE='2024-09-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-17T00:00:00+00:00'
[2024-09-19T18:14:40.615+0000] {pinot_schema_operator.py:35} ERROR - An error occurred: HTTPConnectionPool(host='pinot-controller', port=9000): Max retries exceeded with url: /schemas (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0xffff85a802e0>: Failed to establish a new connection: [Errno 111] Connection refused'))
[2024-09-19T18:14:41.783+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=schema_dag, task_id=submit_schemas, execution_date=20240917T000000, start_date=20240919T181345, end_date=20240919T181441
[2024-09-19T18:14:44.703+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-09-19T18:14:48.662+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
